chpl/chpl-service/src/main/java/gov/healthit/chpl/auth/jwt/JWTConsumerRsaJoseJImpl.java

```diff
@@ -13,41 +13,39 @@
-	JSONWebKey jwk;
-	
-	Logger logger = LogManager.getLogger(JWTConsumerRsaJoseJImpl.class.getName());
-	
-	public Map<String, Object> consume(String jwt) {
-				
-		JwtConsumer jwtConsumer = new JwtConsumerBuilder()
-	    	.setRequireExpirationTime() // the JWT must have an expiration time
-	    	.setAllowedClockSkewInSeconds(Integer.parseInt(env.getProperty("jwtAllowedClockSkew"))) // allow some leeway in validating time based claims to account for clock skew
-	    	.setRequireSubject() // the JWT must have a subject claim
-	    	.setExpectedIssuer(env.getProperty("jwtIssuer")) // whom the JWT needs to have been issued by
-	    	.setExpectedAudience(env.getProperty("jwtAudience")) // to whom the JWT is intended for
-	    	.setVerificationKey(jwk.getKey()) // verify the signature with the public key
-	    	.build(); // create the JwtConsumer instance
-		
-		try
-	    {
-	        //Validate the JWT and process it
-	        JwtClaims jwtClaims = jwtConsumer.processToClaims(jwt);
-	        return jwtClaims.getClaimsMap();
-	    }
-	    catch (InvalidJwtException e)
-	    {
-	    	//log every time an expired / invalid
-	    	// token is used.
-	    	logger.error("Invalid JWT", e);
-	        return null;
-	    }
-	}
+    @Autowired
+    private Environment env;
+
+    @Autowired
+    @Qualifier("RsaJose4JWebKey")
+    JSONWebKey jwk;
+
+    Logger logger = LogManager.getLogger(JWTConsumerRsaJoseJImpl.class.getName());
+
+    public Map<String, Object> consume(String jwt) throws InvalidJwtException {
+
+        JwtConsumer jwtConsumer = new JwtConsumerBuilder().setRequireExpirationTime() // the JWT must have an expiration time
+                .setAllowedClockSkewInSeconds(Integer.parseInt(env.getProperty("jwtAllowedClockSkew"))) // allow some leeway
+                                                                                                        // in validating time
+                                                                                                        // based claims to
+                                                                                                        // account for clock
+                                                                                                        // skew
+                .setRequireSubject() // the JWT must have a subject claim
+                .setExpectedIssuer(env.getProperty("jwtIssuer")) // whom the JWT needs to have been issued by
+                .setExpectedAudience(env.getProperty("jwtAudience")) // to whom the JWT is intended for
+                .setVerificationKey(jwk.getKey()) // verify the signature with the public key
+                .build(); // create the JwtConsumer instance
+
+        try {
+            // Validate the JWT and process it
+            JwtClaims jwtClaims = jwtConsumer.processToClaims(jwt);
+            return jwtClaims.getClaimsMap();
+        } catch (InvalidJwtException e) {
+            logger.error("Invalid JWT", e);
```

Comment:
Does this still log the entire stack trace? If so, can you stop it from doing that? It's not a real error and just makes our logs harder to read